indico/modules/events/abstracts/forms.py

```diff
@@ -396,7 +399,9 @@ class MultiTrackMixin(object):
 
     def __init__(self, *args, **kwargs):
         event = kwargs['event']
-        if abstracts_settings.get(event, 'tracks_required'):
+        self.track_field_disabled = (kwargs.get('abstract') and
+                                     kwargs['abstract'].edit_track_mode != EditTrackMode.both)
+        if abstracts_settings.get(event, 'tracks_required') and not self.track_field_disabled:
```

Comment:
Maybe it's worth having a cleverer way to specify which properties should be proxied in `CallForAbstracts`.